### PR TITLE
Chat api whatsapp template self service

### PR DESCRIPTION
**Pull request by coders-kitchen**

### CODE REVIEW

File: chat-api/v2/openapi.yaml [hunk 1]
```diff
@@ -14,6 +14,32 @@ info:
     - *2-way SMS* : Please contact us to get the details for the 2-way SMS setup
     - *WhatsApp* : A WhatsApp Business Account is required in order to use this channel. Please [request access here](https://www.tyntec.com/form-whatsapp-request-your-access)
 
+    **Adheared web-standards**
+
+    One goal of the API is to follow best practices and standards around http based services.
+
+    A build block of this is to follow RFC standards listed below:
```
Comment (oneiros-de) [inline chat-api/v2/openapi.yaml]: A building block of this is to follow the RFC

File: chat-api/v2/openapi.yaml [hunk 2]
```diff
@@ -267,6 +296,368 @@ paths:
...
+      parameters:
+        - $ref: '#/components/parameters/PhoneNumberQuery'
+      operationId: listAllWhatsAppAccounts
+      summary: List WhatsApp accounts
+      tags:
+        - "WA Account management"
+      description: |
+        Lists all WhatsApp accounts managed by this access credentials.
+
+        In case phone number query parameter is set, only the phone numbers account is listed
```
Comment (oneiros-de) [inline chat-api/v2/openapi.yaml]: If the phone number query parameter is set,

File: chat-api/v2/openapi.yaml [hunk 3]
```diff
@@ -1394,16 +1827,16 @@ components:
...
             Instead use components.
           deprecated: true
           type: array
           items:
             $ref: '#/components/schemas/TemplateParameter'
           example:
-           - default: Mr.
-           - default: Doe
+            - default: Mr.
+            - default: Doe
```
Comment (oneiros-de) [inline chat-api/v2/openapi.yaml]: Check the indentation :-)

File: chat-api/v2/openapi.yaml [hunk 4]
```diff
@@ -2864,9 +3299,584 @@ components:
...
+        - PERSONAL_FINANCE_UPDATE
+        - RESERVATION_UPDATE
+        - SHIPPING_UPDATE
+        - TICKET_UPDATE
+        - TRANSPORTATION_UPDATE
+    LanguageCode:
+      type: string
+      description: Supported language codes, according to [Supported Languages](https://developers.facebook.com/docs/whatsapp/business-management-api/message-templates)
+      enum:
+        - af
```
Comment (oneiros-de) [inline chat-api/v2/openapi.yaml]: Please sort the codes alphabetically :-)

**Comment by coders-kitchen** [comment]
@oneiros-de thank you

**Pull request merged**